src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,34 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/** Deadline form of Task */
+public class Deadline extends Task {
+    private LocalDate by;
+
+    Deadline(String description, LocalDate by) {
+        this(description, false, by);
+    }
+
+    Deadline(String description, boolean isDone, LocalDate by) {
+        super(description, isDone);
+        this.by = by;
+    }
+
+    /**
+     * Returns updated task of subtype: Deadline
+     *
+     * @param isDone New status for the task
+     * @return new Deadline with updated status
```

Comment:
Should there be periods at the end of each of these lines? The coding standard states "Punctuation behind each parameter description".